app/migrations/Version20160926000000.php

```diff
@@ -0,0 +1,122 @@
+  `companyaddress2` varchar(255) DEFAULT NULL,
+  `companycity` varchar(255) DEFAULT NULL,
+  `companystate` varchar(255) DEFAULT NULL,
+  `companyzipcode` varchar(11) DEFAULT NULL,
+  `companycountry` varchar(255) DEFAULT NULL,
+  `companyemail` varchar(255) DEFAULT NULL,
+  `companyphone` varchar(50) DEFAULT NULL,
+  `companyfax` varchar(50) DEFAULT NULL,
+  `companyannual_revenue` float DEFAULT NULL,
+  `companynumber_of_employees` int(11) DEFAULT NULL,
+  `companyscore` int(11) DEFAULT NULL,
+  `companywebsite` varchar(255) DEFAULT NULL,
+  `owner_id` int(11) DEFAULT NULL,
+  `date_added` DATETIME DEFAULT NULL COMMENT '(DC2Type:datetime)',
+  `created_by` int(11) DEFAULT NULL,
+  `created_by_user` varchar(255) DEFAULT NULL,
+  `modified_by` int(11) DEFAULT NULL,
+  `modified_by_user` varchar(255) DEFAULT NULL,
+  `checked_out` datetime DEFAULT NULL COMMENT '(DC2Type:datetime)',
+  `checked_out_by` int(11) DEFAULT NULL,
+  `checked_out_by_user` varchar(255) DEFAULT NULL,
+  `date_modified` datetime DEFAULT NULL COMMENT '(DC2Type:datetime)',
+  `publish_up` datetime DEFAULT NULL COMMENT '(DC2Type:datetime)',
+  `publish_down` datetime DEFAULT NULL COMMENT '(DC2Type:datetime)',
+  `is_published` tinyint(4) DEFAULT NULL,
+  PRIMARY KEY(id),
+  INDEX {$this->prefix}companyname (`companyname`, `companyemail`)
+) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB;
+SQL;
+        $this->addSql($sql);
+        $lead_index = $this->generatePropertyName('company_leads_xref', 'idx', array('lead_id'));
+        $company_index =  $this->generatePropertyName('company_leads_xref', 'idx', array('company_id'));
+        $sql = <<<SQL
+CREATE TABLE {$this->prefix}companies_leads (
+        lead_id INT NOT NULL, 
+        company_id INT NOT NULL, 
+        date_added datetime DEFAULT NULL,
+        manually_added tinyint(4) DEFAULT NULL,
+        manually_removed tinyint(4) DEFAULT NULL,
+        is_primary int(11) NOT NULL DEFAULT 1,
+        INDEX {$lead_index} (lead_id), 
+        INDEX {$company_index} (company_id),
+        PRIMARY KEY(lead_id, company_id)
+        ) 
+        DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB
+SQL;
+
+        $this->addSql($sql);
+
+        $this->addSql("ALTER TABLE {$this->prefix}lead_fields ADD object VARCHAR(255) DEFAULT 'lead'");
+
+        $sql = <<<SQL
+INSERT INTO `{$this->prefix}lead_fields` (`is_published`, `label`, `alias`, `type`, `field_group`, `default_value`, `is_required`, `is_fixed`, `is_visible`, `is_short_visible`, `is_listable`, `is_publicly_updatable`, `is_unique_identifer`, `field_order`, `object`,`properties`) 
+VALUES 
+ (1, 'name', 'companyname', 'text', 'core', NULL, 1, 0, 1, 1, 1, 0, 0, 18, 'company', 'a:0:{}'),
+(1,  'description', 'companydescription', 'textarea', 'professional', NULL, 0, 0, 1, 1, 1, 0, 0, 17, 'company', 'a:0:{}'),
+(1, 'Company Number', 'companynumber', 'text', 'professional', NULL, 0, 0, 1, 1, 1, 0, 0, 16, 'company', 'a:0:{}'),
```

Comment:
Can you remove "Company" from the second column here and on the line below? And give the full proper names in the second column (eg update zipcode to Zip Code)